gcc/rust/parse/rust-parse-impl.h

```diff
@@ -6586,7 +6586,7 @@ Parser<ManagedTokenSource>::parse_path_expr_segment ()
 template <typename ManagedTokenSource>
 AST::QualifiedPathInExpression
 Parser<ManagedTokenSource>::parse_qualified_path_in_expression (
-  bool pratt_parse)
+  const_TokenPtr pratt_parsed_token)
```

Comment:
@philberty I thought you said you wanted to merge the patch containing the Location and not the full Token?

Reply:
Good point I got a bit confused when I raised the PR to review the patch. I will ask on the mailing list again.